src/pr-info.ts

```diff
@@ -358,23 +359,24 @@ async function getPackageInfosEtc(
 
 async function categorizeFile(path: string, contents: (oid?: string) => Promise<string | undefined>): Promise<[string|null, FileInfo]> {
     // https://regex101.com/r/eFvtrz/1
-    const match = /^types\/(.*?)\/.*?[^\/](?:\.(d\.ts|tsx?|md))?$/.exec(path);
+    const match = /^types\/(.*?)\/.*?[^/](?:\.(d\.ts|tsx?|md))?$/.exec(path);
     if (!match) return [null, { path, kind: "infrastructure" }];
     const [pkg, suffix] = match.slice(1); // `suffix` can be null
     switch (suffix || "") {
         case "d.ts": return [pkg, { path, kind: "definition" }];
         case "ts": case "tsx": return [pkg, { path, kind: "test" }];
         case "md": return [pkg, { path, kind: "markdown" }];
-        default:
+        default: {
```

Comment:
Can this be turned off?  (I personally dislike adding vertical space when not needed.)

Reply:
Done :heavy_check_mark: